west.yml

```diff
@@ -22,7 +22,7 @@ manifest:
       remote: memfault
       revision: 1.17.0
     - name: orb-messages
-      revision: 44de6d08fb329da43db9276866d51effc6595fcf
+      revision: e38065ada67c69bbe9d4aab3849c58f6461c3835
```

Comment:
@sri9311 I updated the revision with the PR: https://github.com/worldcoin/orb-messages/pull/44 

let's merge it and track the new commit SHA in that PR